packages/webapp-libs/webapp-core/src/components/buttons/button/button.component.tsx

```diff
@@ -1,37 +1,58 @@
-import { ButtonHTMLAttributes, ReactNode } from 'react';
-import styled, { ThemeProvider } from 'styled-components';
+import { Slot } from '@radix-ui/react-slot';
+import { type VariantProps, cva } from 'class-variance-authority';
+import { ReactNode } from 'react';
+import * as React from 'react';
 
-import { Container, Icon } from './button.styles';
-import { ButtonColor, ButtonSize, ButtonTheme, ButtonVariant } from './button.types';
+import { cn } from '../../../lib/utils';
+import { Icon } from './button.styles';
 
-export type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
-  variant?: ButtonVariant;
-  size?: ButtonSize;
-  color?: ButtonColor | string;
-  icon?: ReactNode;
-  fixedWidth?: boolean;
-};
+const buttonVariants = cva(
```

Comment:
IMO this should be in the `button.styles.ts` file

Reply:
Very good idea